### PR TITLE
Microsoft module GCC url

### PR DESCRIPTION
**Pull request by Shellyber**
## Status
- [x] Ready

## Related Issues
fixes: [XSUP-37179](https://jira-dc.paloaltonetworks.com/browse/XSUP-37179)

## Description
Fixed some more GCC URLs.

### CODE REVIEW

File: Packs/AzureActiveDirectory/ReleaseNotes/1_3_24.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### Azure Active Directory Identity Protection  (Deprecated)
+
+- Fixed an issue in **MicrosoftApiModule** where the GCC endpoints were incorrect.
```
Comment (ShahafBenYakir) [inline Packs/AzureActiveDirectory/ReleaseNotes/1_3_24.md:6]: Customers don't know what is MicrosoftApiModule. Let's change all rn like this:
```suggestion
- Fixed one of the integration's endpoints which pointed to a wrong top-level domain.
```

**Review by ShahafBenYakir — COMMENTED** [review]
This is ok on my end. @michal-dagan is it ok in your opinion?

**Comment by Shellyber** [comment]
I have verified that this is working with the customer as well.

**Review by michal-dagan — COMMENTED** [review]
Nice, FYI @kobymeir 
Maybe make the release note more accurate like the previous PR?
`- Fixed an issue in **MicrosoftApiModule** where the GCC endpoints were incorrect.`

**Comment by ShahafBenYakir** [comment]
@michal-dagan it is not right to address apimodule as it is not something customers should know. The rest of the message is ok

**Comment by michal-dagan** [comment]
> @michal-dagan it is not right to address apimodule as it is not something customers should know. The rest of the message is ok

yes, but the current release note is not relevant to all of the integrations

**Comment by Shellyber** [comment]
so?? what is it going to be? @michal-dagan @ShahafBenYakir

**Comment by Shellyber** [comment]
@ShahafBenYakir @michal-dagan Updated the release notes - please approve.
Probably will need a force merge here as well

**Review by michal-dagan — APPROVED** [review]

**Pull request merged**